readthedocs/core/history.py

```diff
@@ -66,13 +66,11 @@ class ExtraFieldsHistoricalModel(models.Model):
         _("ID"),
         blank=True,
         null=True,
-        db_index=True,
```

Comment:
Maybe we can use the help from @stsewd here. I don't know _where_ these extra fields are used, and if the usage is intense.